peoplehr/tests/test.bal

```diff
@@ -98,8 +100,10 @@ function testCreateNewEmployee() returns error? {
         Location: "Colombo03",
         Department: "Engineering"
     };
-    OperationStatus response = check baseClient->createNewEmployee(employee);
-    test:assertFalse(response.isError, msg = response.Message);
+    OperationStatus|error response = baseClient->createNewEmployee(employee);
+    if response is error {
+        test:assertFail(response.message());
```

Comment:
We need to log the response from server if the assert fails. Then only we can know what is the error server sent. We can do that by

```
test:test:assertFail(xxx, msg = response.Message);
```

Reply:
`test:assertFail` takes only string msg as the input and, if an error occurs, it shows the message.
![image](https://user-images.githubusercontent.com/25779551/217220616-0abdc012-f9a6-473f-854d-4345684fc212.png)

Reply:
+1. Ack